specification/datadog/resource-manager/readme.ruby.md

```diff
@@ -12,11 +12,33 @@ azure-arm: true
 
 ``` yaml $(ruby) && $(multiapi)
 batch:
+  _ tag: package-2023-01
+  - tag: package-2022-08
   - tag: package-2022-06-01
   - tag: package-2021-03-01
   - tag: package-2020-02-01-preview
 ```
 
+### Tag: package-2023-01 and ruby
+
+These settings apply only when `--tag=package-2023-01 --ruby` is specified on the command line.
+Please also specify `--ruby-sdks-folder=<path to the root directory of your azure-sdk-for-ruby clone>`.
+
+``` yaml $(tag) == 'package-2023-01' && $(ruby)
+namespace: "Azure::Datadog::Mgmt::V2023_01_01"
+output-folder: $(ruby-sdks-folder)/management/azure_mgmt_datadog/lib
+```
+
+### Tag: package-2022-08 and ruby
+
+These settings apply only when `--tag=package-2022-08 --ruby` is specified on the command line.
+Please also specify `--ruby-sdks-folder=<path to the root directory of your azure-sdk-for-ruby clone>`.
+
+``` yaml $(tag) == 'package-2022-08' && $(ruby)
+namespace: "Azure::Datadog::Mgmt::V2022_08_01"
+output-folder: $(ruby-sdks-folder)/management/azure_mgmt_datadog/lib
+```
+
```

Comment:
ruby SDK also gets deprecated for quite a time, we no longer need to update this file any more, please revert it.

Reply:
Should i remove the entire file or just revert my changes?
@ArcturusZhang